AUTHORS

```diff
@@ -3139,5 +3139,187 @@ Shigma <33423008+Shigma@users.noreply.github.com>
+DrunkenPoney <mrquebec13@gmail.com>
+Christoph Tavan <dev@tavan.de>
+Clark Kozak <clarkkozak@protonmail.com>
+Michael Auderer <mike@silverstone.io>
+Linn Dahlgren <admin@opposite.biz>
+Shelley Vohr <codebytere@gmail.com>
+Ikko Ashimine <eltociear@gmail.com>
+Anatoly Korniltsev <korniltsev.anatoly@gmail.com>
+Victor Antonio Barzana Crespo <viticoinf@gmail.com>
+Matthieu Larcher <gitlab@ringabell.org>
+anlex N <1293006794@qq.com>
+ThakurKarthik <iamkarthik08@gmail.com>
+Aastha Gupta <aastha.gupta4104@gmail.com>
+Yohanan Baruchel <yohanan@yad2.co.il>
+Dmitry Gozman <dgozman@gmail.com>
+Daniil Demidovich <ddemidovich@staply.ru>
+Hussaina Begum Nandyala <hexxdump@gmail.com>
+Danny Sonnenschein <my.card.god@web.de>
+Sourav Shaw <shawsourav91@gmail.com>
+H Adinarayana <adharida@in.ibm.com>
+lucasg <lucas.georges@outlook.com>
+Brian 'bdougie' Douglas <ilikerobot@gmail.com>
+Lee, Bonggi <iyabong@gmail.com>
+Momtchil Momtchev <momtchil@momtchev.com>
+Josh Dague <jd@josh3736.net>
+Vincent Boivin <vboivin04@gmail.com>
+ax1 <16510021+ax1@users.noreply.github.com>
+Shubham Parihar <51517103+iShibi@users.noreply.github.com>
+raisinten <raisinten@gmail.com>
+Matthew Francis Brunetti <zenflow87@gmail.com>
+Chris Opperwall <copperwall@gmail.com>
+Takuya Noguchi <takninnovationresearch@gmail.com>
+tyankatsu <frips.ryilsufupe+dev@gmail.com>
+Ben Turner <7623873+ben-turner@users.noreply.github.com>
+Bryan Field <bfield@700software.com>
+krank2me <alejok6@gmail.com>
+masx200 <34191203+masx200@users.noreply.github.com>
+Baruch Odem (Rothkoff) <baruchiro@gmail.com>
+Mattias Runge-Broberg <mattias@runge.se>
+Szymon Marczak <36894700+szmarczak@users.noreply.github.com>
+Dmitry Semigradsky <semigradskyd@gmail.com>
+Ole André Vadla Ravnås <oleavr@gmail.com>
+rickyes <0x19951125@gmail.com>
+Aleksandr Krutko <Oleksandr.Krutko@ibm.com>
+Brian Ingenito <28159742+bingenito@users.noreply.github.com>
+FeelyChau <feely@outlook.com>
+Darcy Clarke <darcy@darcyclarke.me>
+mayank agarwal <mayankagarwal44442@gmail.com>
+woodfairy <d.schmit@m3connect.de>
+Nikola Glavina <glavina.nikola5@gmail.com>
+Rishabh Mehan <darkrishabh@gmail.com>
+Anna Henningsen <anna.henningsen@mongodb.com>
+Andrew Casey <andrew.casey@microsoft.com>
+Anders Kaseorg <andersk@mit.edu>
+Hollow Man <hollowman@hollowman.ml>
+nlf <quitlahok@gmail.com>
+naortedgi <naor.tedgi@gmail.com>
+Narasimha Prasanna HN <yogaimpression@gmail.com>
+Zijian Liu <Lxxyxzj@gmail.com>
+inokawa <48897392+inokawa@users.noreply.github.com>
+Michael Bashurov <accounts@saitonakamura.me>
+Moshe vilner <shootermv@gmail.com>
+Nicolai Stange <nicstange@gmail.com>
+kai zhu <kaizhu256@gmail.com>
+FrankQiu <iam.frankqiu@gmail.com>
+Rock <arockplaysmc@gmail.com>
+Chinmoy Chakraborty <chinmoy12c@gmail.com>
+Maksym Baranovskyi <mkbaranovskyi@gmail.com>
+Michael Chen <4326639+mcgitty@users.noreply.github.com>
+François-Denis Gonthier <francois-denis.gonthier@opersys.com>
+Dr <dr@dr.run>
+linkgoron <linkgoron@gmail.com>
```

Comment:
@Linkgoron Does this entry look good to you?

Reply:
I'd prefer Nitzan Uziely <linkgoron@gmail.com> if possible.

Reply:
OK, I think I got it right this time.